src/opts/config.rs

```diff
@@ -83,15 +90,25 @@ pub struct Config {
 }
 
 impl Config {
-    pub fn load() -> anyhow::Result<Self> {
-        let config_dir = dirs::config_dir().context("Failed detecting config dir")?;
+    pub fn load_from_file(path: &Path) -> anyhow::Result<Self> {
+        let config_content = read_to_string(path).context(format!(
+            "Failed to read configuration file at '{path}'",
+            path = path.display()
+        ))?;
```

Comment:
```suggestion
        let config_content = read_to_string(path).context(format!(
            "Failed to read configuration file at '{}'",
            path.display(),
        ))?;
```

I think it's clearer to just use the typical interpolation style here